Sparkle/SUTouchBarButtonGroup.m

```diff
@@ -0,0 +1,60 @@
+//
+//  SUTouchBarButtonGroup.m
+//  Sparkle
+//
+//  Created by Yuxin Wang on 05/01/2017.
+//  Copyright © 2017 Sparkle Project. All rights reserved.
+//
+
+#import "SUTouchBarButtonGroup.h"
+
+@implementation SUTouchBarButtonGroup
+
+@synthesize buttons = _buttons;
+
+- (instancetype)initByReferencingButtons:(NSArray<NSButton *> *)buttons
```

Comment:
> It may be looks like

I think that's good.

> But it conflicts with the property buttons. It may need to three properties for holding three buttons.

Indeed. Although, there is no need to add properties if they're currently being unused by Sparkle, so you may be able to get away with one property.

I think that the layout cares is a good enough reason. It's not as simple as "right to left" -- padding differs in a certain spot, and one button has a return mapping.